CHANGELOG.md

```diff
@@ -5,6 +5,7 @@
 - Switch from `Flask-restplus` to its fork `Flask-rest-x` [#165](https://github.com/etalab/udata-front/pull/165)
 - Add CaptchEtat integration [#159](https://github.com/etalab/udata-front/pull/159)
   - new environment variables : CAPTCHETAT_BASE_URL, CAPTCHETAT_OAUTH_BASE_URL, CAPTCHETAT_CLIENT_ID and CAPTCHETAT_CLIENT_SECRET
+- Improve search history [#162](https://github.com/etalab/udata-front/pull/162)
```

Comment:
You will need to move this entry to the new current section.

Reply:
Done :)